index.html

```diff
@@ -0,0 +1,182 @@
+            <p>HUNIDITY</p>
+            <p>49.0</p>
+          </div>
+          <div class="amountToProduce btOV-child">
+            <span class="iconify icon-btOverView" data-icon="fa-solid:faucet"> </span>
+            <p>AMOUNT TO PRODUCE</p>
+            <p>6000</p>
+          </div>
+          <div class="acceptableProducts btOV-child">
+            <span class="iconify icon-btOverView" data-icon="bx:bx-check"> </span>
+            <p>ACCEPTABLE PRODUCTS</p>
+            <p>223</p>
+          </div>
+          <div class="vibration btOV-child">
+            <span class="iconify icon-btOverView" data-icon="zmdi:vibration"></span>
+            <p>VIBRATION</p>
+            <p>0.0</p>
+          </div>
+          <div class="productsPerMin marked btOV-child">
+            <span class="iconify icon-btOverView" data-icon="cil:speedometer"> </span>
+            <p>PRODUCTS PER MINUTE</p>
+            <p>300</p>
+          </div>
+          <div class="DefectProducts btOV-child">
+            <span class="iconify icon-btOverView" data-icon="bi:x"> </span>
+            <p>DEFECT PRODUCTS</p>
+            <p>43</p>
+          </div>
+        </div>
+
+        <!-- Inventory of the beer machine -->
+        <div class="inventory-st">
+          <div class="inventory-top">
+            <p>INVENTORY STATUS</p>
+          </div>
+          <div class="inventory-container">
+            <div class="BARLEY inventory-inner">
+              <div></div>
+              <div></div>
+              <div class="BARLEY inventory-overlay">
+                <p>BARLEY</p>
+                <p>0.100KG</p>
+              </div>
+            </div>
+            <div class="MALT inventory-inner">
+              <div></div>
+              <div></div>
+              <div class="MALT inventory-overlay">
+                <p>MALT</p>
+                <p>0.100KG</p>
+              </div>
+            </div>
+            <div class="HOPS inventory-inner">
+              <div></div>
+              <div></div>
+              <div class="HOPS inventory-overlay">
+                <p>HOPS</p>
+                <p>0.100KG</p>
+              </div>
+            </div>
+            <div class="WHEAT inventory-inner">
+              <div></div>
+              <div></div>
+              <div class="WHEAT inventory-overlay">
+                <p>WHEAT</p>
+                <p>0.100KG</p>
+              </div>
+            </div>
+            <div class="YEAST inventory-inner">
+              <div></div>
+              <div></div>
+              <div class="YEAST inventory-overlay">
+                <p>YEAST</p>
+                <p>0.100KG</p>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <!-- maintain info -->
+        <div class="maintains">
+          <div id="maintains-container">
+            <div></div>
+            <div></div>
+          </div>
+          <div id="maintains-bottom">
+            <p>100%</p>
```

Comment:
100% what? I know it's just a placeholder, but we need to think about how to use the percentage at the maintenance bar. Does 100% mean that 100% of the machine needs maintenance or the other way around?

Reply:
yes 100% was just because I thought it was nice, 
we can make it as a overlay like the inventory stuff? 
and then add it to the bottom so you can se it is what is left maybe?

Reply:
Not quite sure I understand. Could you elaborate?

Reply:
![image](https://user-images.githubusercontent.com/55736468/99541004-95414e00-29b0-11eb-93ab-5bae99a416f8.png)

you can see % is a part of the bar now 
so you know where it belongs